src/ui/components/CustomButtons/CodeActionButton.jsx

```diff
@@ -6,14 +8,50 @@ import {
   CodeIcon,
   CopyIcon,
   TerminalIcon,
+  KeyIcon,
 } from '@primer/octicons-react';
-import React, { useState } from 'react';
+
+const API_BASE = import.meta.env.VITE_API_URI ?? '';
```

Comment:
I wonder if there are edge cases where the `VITE_API_URI` is not present? Other parts of the app seem to have a fallback value (current origin):

```ts
const baseUrl = import.meta.env.VITE_API_URI
  ? `${import.meta.env.VITE_API_URI}`
  : `${location.origin}`;
```

Reply:
I've added the same comment to the upstream PR review, so feel free to ignore this!